### ISSUE TITLE
prototype notification service with CIMI

### ISSUE THREAD

**Issue by cjdcordeiro** [slipstream/SlipStreamServer#1676]

**Comment by loomis** [comment]
@cjdcordeiro Can you summarize the desired features and requirements in this ticket?  This will be easier for me than trying to remember the previous discussions.

**Comment by cjdcordeiro** [comment]
@loomis as a summary of what was described [here](https://github.com/mF2C/mF2C/issues/75), what's needed is mostly the ability to get notified when:
 - a new CIMI resource X is added/deleted (and perhaps modified)

The expected notification should identify which kind of event happened (resourceLeft, newResource, etc.) and contain the body of the corresponding resource
